mp/src/game/client/client_momentum.vpc

```diff
@@ -312,6 +312,8 @@ $Project "Client (Momentum)"
             $File   "momentum\c_te_shotgun_shot.cpp"
             $File   "momentum\fx_mom_weaponfx.cpp"
             $File   "momentum\fx_mom_impacts.cpp"
+            $File   "momentum\fx_mom_explosions.cpp"
+            $File   "momentum\mom_fx_particleeffect.cpp"
```

Comment:
Why is this file breaking naming convention with the others? Rename it to be `fx_mom_particles.cpp` instead.